docs/codacy-api/api-tokens.md

```diff
@@ -22,36 +22,36 @@ The sections below provide detailed instructions on how to generate and revoke A
 
 ## Generating and revoking account API tokens {: id="account-api-tokens"}
 
-To generate an account API token:
+You can create new account API tokens programmatically using the Codacy API endpoint [createUserApiToken](https://api.codacy.com/api/api-docs#createuserapitoken){: target="_blank"} or using the Codacy UI:
 
 1.  Open your account, tab **Access management**.
 
 1.  Click the button **Create API token** under **Account API tokens**.
 
+    !!! tip
+        You can create multiple account API tokens. This can be useful to have a more flexible control by revoking only a specific token.
+
     ![Creating an account API token](images/codacy-api-tokens-account.png)
 
 To revoke an account API token, click the "X" next to the token. After this, all applications or services using that token to access the Codacy API will fail to authenticate and will receive the reply `{"error":"not found"}`.
 
-!!! tip
-    You can create multiple account API tokens. This can be useful to have a more flexible control by revoking only a specific token.
-
 ## Generating and revoking project API tokens {: id="project-api-tokens"}
 
-To generate a project API token:
+You can create new account API tokens programmatically using the Codacy API endpoint [createProjectApiToken](https://api.codacy.com/api/api-docs#createprojectapitoken){: target="_blank"} or using the Codacy UI:
```

Comment:
```suggestion
You can create new project API tokens programmatically using the Codacy API endpoint [createProjectApiToken](https://api.codacy.com/api/api-docs#createprojectapitoken){: target="_blank"} or using the Codacy UI:
```

Reply:
I'm glad you spotted this quintessential copy+paste error! :bow: :sweat_smile: